libraries/aws_vpcs.rb

```diff
@@ -0,0 +1,42 @@
+class AwsVpcs < Inspec.resource(1)
+  name 'aws_vpcs'
+  desc 'Verifies settings for AWS VPCs in bulk'
+  example '
+    describe aws_vpcs do
+      it { should exist }
+    end
+  '
+
+  # Underlying FilterTable implementation.
+  filter = FilterTable.create
+  filter.add_accessor(:entries)
+        .add(:exists?) { |x| !x.entries.empty? }
+  filter.connect(self, :access_key_data)
+
+  def access_key_data
```

Comment:
This is probably copypasta from access_keys.  You should select a method name here that is specific to vpcs (`vpc_data`, perhaps)